jupyterlab/labbox_ephys_widgets_jp/src/initializeHitherForJpWidgetView.tsx

```diff
@@ -0,0 +1,64 @@
+import { WidgetModel } from '@jupyter-widgets/base';
```

Comment:
This code has been separated out because it is shared between sorting view and workspace view. It handles the hither communication between client/server.